lib/tokenize.h

```diff
@@ -538,8 +538,8 @@ class CPPCHECKLIB Tokenizer {
 
     /**
      * Modify strings in the token list by replacing hex and oct
-     * values. E.g. "\x61" -> "a" and "\000" -> "\0"
-     * @param source The string to be modified, e.g. "\x61"
+     * values. E.g. "\\x61" -> "a" and "\000" -> "\0"
```

Comment:
I think this is a mistake. If we see `"\\x61"` in the code then that should not be converted to `"a"` as far as I see.

Reply:
This is escaping for doxygen so it doesn't think it is the command `\x` aka `@x`. Doxygen commands can be prefixed with either `\` or `@` - so `@param` could also be written as `\param`.

Reply:
Unless you're using a GUI which "translates" on the fly you have to decide if the comment should be formatted for your eyes or for doxygen.
Maybe in some cases one may switch back use a non-doxygen comment...

Reply:
I think it's better that the comment is correct. If you can workaround it in some other way that is better otherwise it seems we can't enable these warnings.

Reply:
You cannot just ignore the syntax just because you don't like it. And if doxygen ever adds a `x` command it will be totally wrong.

It only applies to `\` or `@` followed by a character. And it's just in two places so I don't think this will be a major issue.